code/go/0chain.net/blobbercore/handler/storage_handler.go

```diff
@@ -26,6 +26,10 @@ const (
 
 	DOWNLOAD_CONTENT_FULL  = "full"
 	DOWNLOAD_CONTENT_THUMB = "thumbnail"
+	PAGE_LIMIT             = 100 //100 rows will make upto 100 KB
+	UPDATED                = "updated"
+	DELETED                = "deleted"
+	REGULAR                = "regular"
```

Comment:
What is the need for these constants, we can directly use the string name in the switch?

Reply:
Done